src/rest/v1/openapi.yml

```diff
@@ -287,3 +322,31 @@ paths:
                     type: array
                     items:
                       $ref: '#/components/schemas/Shift'
+  /rate-worker:
```

Comment:
This API is deviating from REST standards. Verbs are not recommended for REST apis

Reply:
Yeah...missed this.